.github/workflows/ci.yml

```diff
@@ -191,6 +192,10 @@ jobs:
       env:
         SKIP_TEST: true
       run: sbt 'project transformerKinesis' assembly
+    - name: Build transformer-pubsub jar
+      env:
+        SKIP_TEST: true
+      run: sbt 'project transformerPubsub' assembly
```

Comment:
We could replace L175-198 with a single step

```
sbt \
'project redshiftLoader; assembly' \
'project snowflakeLoader; assembly' \
'project databricksLoader; assembly' \
'project transformerBatch; assembly' \
'project transformerKinesis; assembly' \
'project transformerPubsub; assembly'
```

assuming that we don't want simpler `sbt assembly` not to wait on building all projects' fatjars